tests/util/kube_utils.go

```diff
@@ -127,6 +127,15 @@ func NamespaceDeleted(n string, kubeconfig string) (bool, error) {
 	return false, err
 }
 
+// ValidatingWebhookConfigurationDeleted check if a kubernetes ValidatingWebhookConfiguration is deleted
+func ValidatingWebhookConfigurationDeleted(name string, kubeconfig string) (bool, error) {
```

Comment:
Only the boolean `deleted` value is checked and the `err` return value is always ignored. Consider removing the error unless the caller needs to do something with the error value.

Reply:
done